package.json

```
@@ -6,7 +6,9 @@
   "scripts": {
     "lint": "prettier -c .",
     "format": "prettier -w .",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "bundle": "redocly bundle -d --output dist openapi.yaml",
```

```suggestion
    "bundle": "redocly bundle -d --output dist/hookdeck.yaml openapi.yaml",
```

Oh, overlooked this - can we toss this in a dist directory instead of naming it `dist.yaml`?

---

yup!